### ISSUE TITLE
"Light" blocks block smeltery from working

### ISSUE THREAD

**Issue by disappeaTed** [SlimeKnights/TinkersConstruct#5241]
### Minecraft Version

1.18.2

### Forge Version

40.2.21

### Mantle Version

1.9.50

### Tinkers' Construct Version

3.7.1.155

### Describe your issue

TL;DR: Smeltery does not recently added "Light" blocks inside it. I consider this a bug, because mods like [Dynamic Lights](https://www.curseforge.com/minecraft/mc-mods/tschipcrafts-dynamic-lights-mod) work by placing invisible "Light" blocks.

1) Normally placed light (no mods):
![image](https://github.com/SlimeKnights/TinkersConstruct/assets/157877453/7e75dd9c-d160-4a3d-89aa-f34e6af3e993)

2) Blaze emitting light blocks (Dynamic Lights mod):
![image](https://github.com/SlimeKnights/TinkersConstruct/assets/157877453/9f485fc2-f4d4-4214-bf69-28a4b8829699)

### Other mods

[Dynamic Lights](https://www.curseforge.com/minecraft/mc-mods/tschipcrafts-dynamic-lights-mod).

### Tried reproducing with just Tinkers?

Yes

### Performance Enchancers

None of the above

### Searched for known issues?

Searched open issues, Searched closed issues

**Comment by ShinyPorygon** [comment]
Yes the smeltery gets invalidated with any block put inside it. Use a dynamic light mod that doesn't do this such as 
https://www.curseforge.com/minecraft/mc-mods/dynamiclights-reforged no one has had problems with this light mod.

**Comment by KnightMiner** [comment]
I'm going to reopen this as while that dynamic light mod is doing things in a bad way, I see the appeal of light blocks in smelteries for creative players and it might give us an opportunity to make the smeltery behavior a bit more resilient to such weird cases.

**Comment by KnightMiner** [comment]
3.9.2.34 allows light blocks in the smeltery, and adds a tag for other non-standard air blocks. I don't currently plan to backport this to 1.19, so closing.
